test/gst-msdk/vpp/csc.py

```diff
@@ -0,0 +1,54 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+
+spec = load_test_spec("vpp", "csc")
+
+@slash.requires(have_gst)
+@slash.requires(*have_gst_element("msdk"))
+@slash.requires(*have_gst_element("msdkvpp"))
+@slash.requires(*have_gst_element("checksumsink2"))
+@slash.requires(using_compatible_driver)
+@slash.parametrize(*gen_vpp_csc_parameters(spec))
+@platform_tags(VPP_PLATFORMS)
+def test_default(case, csc):
+  params = spec[case].copy()
+  params.update(
+    csc = csc, mcscu = mapformatu(csc),
+    mformat = mapformat(params["format"]))
+  params["ofile"] = get_media()._test_artifact(
+    "{}_{format}_csc_{csc}_{width}x{height}"
+    ".yuv".format(case, **params))
+
+  call(
+    "gst-launch-1.0 -vf filesrc location={source} num-buffers={frames}"
+    " ! rawvideoparse format={mformat} width={width} height={height}"
+    " ! msdkvpp hardware=true ! videoconvert ! video/x-raw,format={mcscu}"
```

Comment:
This is still wrong.  msdkvpp is doing nothing here.  CSC is being done by videoconvert which is SW CSC.  You need to do CSC with msdkvpp

Reply:
I follow the full test spec to design this, I will investigate it.

Reply:
The test spec is probably wrong.  @FocusLuo any comment?